Expensify/App
# Hide unread messages

HOLD - till this auth [PR](https://github.com/Expensify/Server-Expensify/pull/4818) is live.

# Fixes
https://github.com/Expensify/ReactNativeChat/issues/330

# Tests
1. Open chat UI as expensify user1. Confirm you see the pinned reports and nothing else.
1. Create a new expensify account and have that user (say user2) also open chat and send a DM to user1.
1. Confirm user1 sees the new chat pop up in LHN in realtime and its unread. (sometimes the first message doesn't show up but i think thats related to another issue that i'm not diving in here)
1. Don't click on the chat and have user1 refresh their page. Confirm the chat still shows up in LHN and its still unread.
1. Click on the chat. Send a message. Click on another pinned report. Confirm the unpinned message vanishes.
